05/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" href="./styles/desktop.css" media="(min-width: 800px)" >
+</head>
+<body>
+    <header class="row header">
+        <div class="container header__container">
+            <h1 class="header__logo cell"></h1>
+            <button class="hamburger"></button>
+            <nav class="nav cell">
+                <ul class="nav__menu">
+                    <li class="menu__item"><a href="">One</a></li>
+                    <li class="menu__item"><a href="">Two</a></li>
+                    <li class="menu__item"><a href="">Three</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <main class="main">
+        <section class="main__section-one row">
+            <div class="container content__container-between">
```

Comment:
Raczej użyłbym modyfikatora więc byłoby coś takiego:  `container content__container content__container---between`.
Gdzie `content__container` zawiera wspólne cechy dla tego typu elementów oraz  `content__container---between` szczególne dla `between`.

Tutaj też uwaga, że skoro mamy element `content__container` to gdzieś w hierarchii winien być blok tj. `content`